### ISSUE TITLE
Feature Request: Release Notes

### ISSUE THREAD

**Issue by ZV00** [AerialScreensaver/Aerial#2]
Hey @glouel 

Was thinking some further thoughts today and realized it would be awesome to have the ability to easily access the latest release notes from Aerial Updater. This may be more of a "pro user" type feature, but I know I would find it very helpful. 

Thanks!

**Comment by glouel** [comment]
Agreed. Short term I can open them in a browser, there may be a better way to do this to grab them and put them in a system notification for example, I'll investigate !

**Comment by glouel** [comment]
We have release notes now via Sparkle 🍾
